bundles/tools.vitruv.applications.umljava/src/tools/vitruv/applications/umljava/java2uml/JavaToUmlMethod.reactions

```diff
@@ -355,27 +355,27 @@ routine changeUmlReturnType(java::Method jMeth, java::TypeReference jType) {
 //=========================================== General
 //=========================================== 
 
-reaction JavaElementVisibilityChanged {
-    after element java::Modifier created and inserted in java::AnnotableAndModifiable[annotationsAndModifiers]
+reaction JavaElementVisibilityChangedAfterInsertion {
+    after element java::Modifier inserted in java::AnnotableAndModifiable[annotationsAndModifiers]
     with newValue instanceof Public || newValue instanceof Private || newValue instanceof Protected
-    call changeUmlNamedElementVisibility(affectedEObject, newValue)
+    call changeUmlNamedElementVisibility(affectedEObject)
 }
 
-reaction JavaElementMadePackagePrivate {
-    after element java::Modifier deleted and removed from java::AnnotableAndModifiable[annotationsAndModifiers]
+reaction JavaElementMadeVisibilityChangedAfterRemoval {
+    after element java::Modifier removed from java::AnnotableAndModifiable[annotationsAndModifiers]
     with oldValue instanceof Private
     || oldValue instanceof Public
     || oldValue instanceof Protected
-    call changeUmlNamedElementVisibility(affectedEObject, null)
+    call changeUmlNamedElementVisibility(affectedEObject)
 }
 
-routine changeUmlNamedElementVisibility(java::AnnotableAndModifiable jElem, java::Modifier mod) {
-    match {
+routine changeUmlNamedElementVisibility(java::AnnotableAndModifiable jElem) {
+	match {
         val uElem = retrieve uml::NamedElement corresponding to jElem tagged with ""
     }
```

Comment:
The indentation is inconsistent.

Reply:
Impressive you noticed that. I fixed indentation for the entire file and removed unnecessary semicolons.